command/namespace_apply.go

```diff
@@ -33,15 +33,19 @@ Apply Options:
 
   -description
     An optional description for the namespace.
+
+  -enabled-task-drivers
+    A comma separated list of allowed task drivers.
 `
 	return strings.TrimSpace(helpText)
 }
 
 func (c *NamespaceApplyCommand) AutocompleteFlags() complete.Flags {
 	return mergeAutocompleteFlags(c.Meta.AutocompleteFlags(FlagSetClient),
 		complete.Flags{
-			"-description": complete.PredictAnything,
-			"-quota":       QuotaPredictor(c.Meta.Client),
+			"-description":          complete.PredictAnything,
+			"-quota":                QuotaPredictor(c.Meta.Client),
+			"-enabled-task-drivers": complete.PredictAnything,
```

Comment:
The RFC suggested using this because it made for a reasonable initial implementation, but after some internal discussion I think we're going to want to have a namespace spec like we do for quotas and volumes. Otherwise we end up having to go through a deprecation process on this flag later.

The [`quota apply`](https://github.com/hashicorp/nomad/blob/main/command/quota_apply.go#L83-L118) command has some decent example code of this. The spec can be the serialized version of the existing `api.Namespace` with the new `Capabilities` field.

Reply:
How to handle the additional flags? Or rather is there is a file passed simply ignore them?

Reply:
Maybe on the code path where we have a file we can ignore the flags but report a warning to the user that the file took precedence?